en/core-api/advanced-features.md

```diff
@@ -784,18 +997,22 @@ curl -X POST \
 }
 ```
 
-#### Two Clicks Initials Open 3DS Authentication Page
-As part of API response, we now have `redirect_url`. It should be opened (displayed to customer) using [MidtransNew3ds JS library](https://api.midtrans.com/v2/assets/js/midtrans-new-3ds.min.js) on merchant's website frontend.
+?>***Note***: The `transaction_status` is *Pending*.
+
+<!-- tabs:end -->
+
+#### Opening 3DS Authentication Page for the First Transaction
+To open 3DS authentication page on merchant frontend, display the `redirect_url` retrieved from Charge API Response for the First Transaction. The redirect URL is displayed using `MidtransNew3ds.authenticate` or `MidtransNew3ds.redirect` function in [MidtransNew3DS JS library](https://api.midtrans.com/v2/assets/js/midtrans-new-3ds.min.js).
+
+ When the customer completes the transaction, the *Transaction Status* changes from *Pending* to *Capture*.
+
+ For more details, refer to [Open 3DS Authenticate Page JS Implementation Step](/en/core-api/credit-card#open-3ds-authenticate-page-js-implementation).
```

Comment:
1. Link: `/en/core-api/credit-card#open-3ds-authenticate-page-js-implementation` no longer point to the correct Header of the destination page, because you renamed the header on the credit-card page. Please update with the correct header name/url.

2. Also the correct format should be 
> credit-card.md#open-3ds-authenticate-page-js-implementation

Not
> credit-card#open-3ds-authenticate-page-js-implementation


This apply to all other links which point to specific header. (Links that have `#`).